src/main/java/net/backupcup/mcde/util/EnchantmentUtils.java

```diff
@@ -197,7 +226,7 @@ public void onSlotUpdate(ScreenHandler handler, int slotId, ItemStack stack) {
                 }
                 context.run((world, pos) -> {
                     var server = world.getServer();
-                    var serverPlayer = server.getPlayerManager().getPlayer(player.getUuid());
+                    var serverPlayer = Optional.ofNullable(server.getPlayerManager().getPlayer(player.getUuid()));
```

Comment:
as with `PlayerEntity playerEntity` vs `PlayerEntity player`, Yarn uses `ServerPlayerEntity serverPlayerEntity`. Regardless of whether or not you're using `var`, the nomenclature should be consistent with the mappings you're using.